src/devices/Gpio/README.md

```diff
@@ -1,80 +1,34 @@
-# Sunxi GPIO Driver for .NET
+# GpioDriver for other boards
 
-**sunxi** represents the family of ARM SoCs from Allwinner Technology. This project contains a **full function(PULL-UP, PULL-DOWN)** generic GPIO driver `SunxiDriver` for Allwinner SoCs and some special GPIO drivers like `OrangePiZeroDriver`, `OrangePiLiteDriver`, `OrangePiLite2Driver`.
+This project contains some **full function(PULL-UP, PULL-DOWN)** generic GPIO drivers, and it can provide faster GPIO access.
 
-## Getting started
+## Catalogue
 
-### Special GPIO driver: `OrangePiZeroDriver`
+* For Allwinner SoCs: [SunxiDriver](Drivers/Sunxi/README.md)
+* For Rockchip SoCs: [RockchipDriver](Drivers/Rockchip/README.md)
 
-```C#
-// For Orange Pi Zero
-using GpioController gpio = new GpioController(PinNumberingScheme.Board, new OrangePiZeroDriver());
+## Benchmarks
 
-// Open the GPIO pin.
-gpio.OpenPin(7);
-// Set the pin mode.
-gpio.SetPinMode(7, PinMode.InputPullUp);
-// Read current value of the pin.
-PinValue value = gpio.Read(7);
-```
+The test uses different GPIO drivers to quickly switch the state of GPIO, and uses an oscilloscope to measure the average frequency of GPIO externally.
 
-### Generic GPIO driver: `SunxiDriver`
+### SunxiDriver
 
-```C#
-// Beacuse this is a generic driver, the pin scheme can only be Logical.
-// The base addresses can be found in the corresponding SoC datasheet.
-using GpioController gpio = new GpioController(PinNumberingScheme.Logical, new SunxiDriver(cpuxPortBaseAddress: 0x01C20800, cpusPortBaseAddress: 0x01F02C00));
+Benchmarking with **Orange Pi Zero**, select GPIO 6 (Logical). The operating system is Armbian buster, Linux kernel version is 5.10.16, and .NET version is 5.0.3.
 
-// Convert pin number to logical scheme.
-int pinNumber = SunxiDriver.MapPinNumber(portController: 'A', port: 10);
-gpio.OpenPin(pinNumber);
-gpio.SetPinMode(pinNumber, PinMode.Output);
-// Write a value to the pin.
-gpio.Write(pinNumber, PinValue.High);
-```
+| Drivers| Language | Library Version | Test Date | Average Frequency |  |
+| :-: | :-: | :-: | :-: | :-: | :-: |
+| SunxiDriver | C# | - | 2020-02-20 | 185 KHz | <img src="imgs/SunxiDriver/sunxi.jpg" height="120"/> |
```

Comment:
Maybe better to resize properly the image and not use HTML code inside the Markdown. Once the images are resized, you can then use the ```![decription](./imgs/restofuri)``` pattern

Reply:
I think it's ok like this, because we want to have this nicely displayed in a table for comparison while not loosing the full resolution of the image (click the image to see it larger).